.env.hoodi

```
@@ -0,0 +1,12 @@
+export REACT_APP_IS_MAINNET=false
+export REACT_APP_CONTRACT_ADDRESS=0x4242424242424242424242424242424242424242
```

Shouldn't this be:

```suggestion
export REACT_APP_CONTRACT_ADDRESS=0x00000000219ab540356cBB839Cbe05303d7705Fa
```

As per https://github.com/eth-clients/hoodi/blob/08dd242abdb1f93026453bf8e63e6bba1c7b1bc1/metadata/genesis.json#L31